compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -100,6 +111,8 @@ void buildGlobalOptimizationPassPipeline(
       .addPass(createConvert1X1FilterConv2DToMatmulPass);
   mainPassManager.addPass(createEraseUnusedLinalgOperands());
 
+  mainPassManager.addPass(createTransposeMatmulPass(clTransposeMatmul));
```

Comment:
I think it would be best to not create the pass if `clTransposeMatmul == TransposeMatmulOption::None`. 
I assume there's a cost to setting up and running a pass even if it does nothing.